scripts/listen.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+require(__dirname + '/../' + 'constants');
+
+const requested = process.env.npm_config_collection;
```

Comment:
It might be better to initialize this in constants.js and also provide a default value if  'collection' is not passed in the npm configuration.